distributed/asyncio.py

```diff
@@ -0,0 +1,91 @@
+# flake8: noqa
+
+import asyncio
+from functools import wraps
+
+from tornado.gen import is_coroutine_function
+from tornado.platform.asyncio import BaseAsyncIOLoop
+from tornado.platform.asyncio import to_asyncio_future, to_tornado_future
+
+from .client import Client, Future
+
+from tornado.ioloop import IOLoop
+
+
+def to_asyncio(method):
+    @wraps(method)
+    def convert(*args, **kwargs):
+        return to_asyncio_future(method(*args, **kwargs))
+    return convert
+
+
+class AioLoop(BaseAsyncIOLoop):
+
+    @property
+    def _running(self):
+        """Distributed checks IOLoop's _running property extensively"""
+        return self.asyncio_loop.is_running()
```

Comment:
Yes, that was my question.  (this is a genuine question, not a suggestion).  Rather than wrap things in a custom subclass is it cleaner to make a function like the following:

```python
def is_running(io_loop):
    if isinstance(io_loop, tornado.ioloop.IOLoop):
        return io_loop._running
    else:
        return ...
```